ant-design/ant-design
# chore: next merge feature

- fix(Popconfirm): the title element does not have padding when icon is not set and update jest snapshot (#37384)
- feat: Steps supports items (#37531)
- docs: Tree select data driven (#37622)
- feat:  AutoComplete edit demo to data driven (#37592)
- feat: Badge support colorful custom count style (#37609)
- Tree data driven (#37626)
- feat: select edit demo to data driven (#37601)
- feat: Components under Form support custom disabled to override disabled of Form (#37628)
- feat: add icon to collapsible (#37566)
- feat: bump rc-image version, Image add preview.scaleStep prop (#37340)
- chore: merge master
- chore: fix lint
- chore: fix lint
- test: Update snapshot
- test: Update snapshot
- fix: Nest Drawer default open show order (#37767)
- feat(table): support close fn to close filter dropdown only (#37745)
- docs: fix demo code tsx copy (#37789)
- refactor: Use new rc-dialog (#37790)
- fix: delete node_modules
- docs(Tabs): wrong size type (#37796)
- type: strong type (#37799)
- fix: password should not have value prop during inputing (#37801)
- type: strong type (#37805)
- style: Code optimization (#37804)
- fix: Typography ellipsis percision bug (#37802)
- chore: prefer using @internal annotation over other markups (#37816)
- fix: remove defaultProps (#37815)
- chore: fix prevents this action from running on forks (#37809)
- docs: fix antd site in mobile device (#37819)
- chore: PR template's wording (#37821)
- chore: fix typo customer (#37820)
- fix: delete defaultprops (#37822)
- Fix a typo (#37826)
- chore: fix typo (#37832)
- fix: delete defaultprops (#37827)
- refactor(locale): refine signature (#37824)
- docs: add  4.23.4 changelog (#37833)
- chore(deps): update dependency eslint-plugin-unicorn to v44 (#37837)
- refactor: delete defaultprops (#37834)
- test: replace Popconfirm part test with testing lib (#37703)
- test: replace Calendar part test with testing lib (#37704)
- docs(Tabs): replace `TabItem` with `TabItemType` (#37846)
- fix: Textarea blink on first mount (#37847)
- refactor: delete defaultprops (#37853)
- fix: update RegExp (#37856)
- feat: add missing pt_PT translations (#37857)
- fix: Fix Russain translation for Transfer.selectInvert (#37858)
- chore: delete Enzyme (#37854)
- fix: input search border style in RTL (#37865)
- fix: signatures for Upload (#37861)
- feat: Notification support for data-testid (#37710)
- fix: when we apply style resize: none, TextArea should be not resizable when showCount is applied (#37855)
- chore(deps-dev): bump @types/gtag.js from 0.0.11 to 0.0.12 (#37870)
- feat: Performance improvement on React useState (#37875)
- ci: add minimum GitHub token permissions for workflow (#37845)
- chore(deps): update dependency react-dnd to v16 (#37840)
- test: Use SSR for testing demo (#37890)
- test: rename  cascader test case name (#37891)
- chore(deps): update dependency eslint-plugin-react to v7.31.8 (#37887)
- docs: sync english docs (#37892)
- style: Input.Password support style={{ fontSize }} (#37898)
- docs: FAQ about dropdownRender (#37906)
- docs: fix site locale (#37911)
- docs: add image.preview.forceRender props (#37808)
- refactor: CC => FC (#37886)
- test: Update table test sleep -> useFakeTimers (#37914)
- docs: changelog for 4.23.5 (#37917)
- chore: bug mark of rc-textarea
- fix: tsc error (#37932)
- fix: Tabs overflow calculation (#37937)
- fix: Upload disabled logic
- test: Update snapshot



[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link



### 💡 Background and solution



### 📝 Changelog



| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
